### PR TITLE
Ignore .cargo/insta.yaml

### PR DESCRIPTION
**Pull request by essiene**
Ignore .cargo/insta.yaml so folks to customize this locally.

See https://insta.rs/docs/settings/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@ result
 *.pending-snap
 *.snap*
 !cli/tests/cli-reference@.md.snap
+.cargo/insta.yaml
```
Comment (yuja) [inline .gitignore:11]: Can you add some comment saying this is per-user configuration?

Comment (yuja) [inline .gitignore:11]: You mean `.config/insta.yaml`?
https://insta.rs/docs/settings/#tool-config-file

Comment (essiene) [reply .gitignore]: Done.

Comment (essiene) [reply .gitignore]: Whoops! Fixed. Thanks

**Review by yuja — APPROVED** [review]

**Pull request merged**